src/Components/Server/src/DependencyInjection/ComponentServiceCollectionExtensions.cs

```diff
@@ -23,6 +24,7 @@ public static class ComponentServiceCollectionExtensions
         /// <returns>The <see cref="IServiceCollection"/>.</returns>
         public static IServiceCollection AddRazorComponents(this IServiceCollection services)
         {
+            services.AddLogging();
```

Comment:
Doesn't hosting set this up?

Reply:
I was getting errors for this. It should be harmless to do.